.env.example

```diff
@@ -1,5 +1,8 @@
-STORYBOOK_AUTH_TOKEN=
-# Env var STORYBOOK_INSTANCE_ID should be set to the ID of some instance in Sentinel Hub
+CLIENT_ID=
+CLIENT_SECRET=
+
+# Env var INSTANCE_ID should be set to the ID of some instance in Sentinel Hub
 # Configurator, which contains all the layers listed below it.
-STORYBOOK_INSTANCE_ID=
-STORYBOOK_S2L2A_LAYER_ID=
+INSTANCE_ID=
+S2L2A_LAYER_ID=
+S1GRD_LAYER_ID=
```

Comment:
Are you sure this works? As per [the docs](https://storybook.js.org/docs/configurations/env-vars/), Storybook only imports those env vars that start with `STORYBOOK_` - at least this is how I read them. I agree it is annoying though,as we will either have all vars inappropriately named (they are not Storybook-specific), or we will need to duplicate them.

Alternatively, we could change `package.json` so that all env vars are exported with `STORYBOOK_` prefix, but I'm not sure if this is too much "magic"?

Reply:
I have no idea why, but it works for me the way it is now. Can you maybe also try?

Reply:
Interesting, so it does... :)

Sorry for the noise, resolving.